install.d/50-dracut.install

```diff
@@ -71,7 +71,6 @@ done
 
 # shellcheck disable=SC2046
 dracut -f \
-    --add-confdir hostonly \
```

Comment:
Why remove this? IMO it makes sense to have a config directory dedicated to running dracut from `install.d`. It's debatable whether that config dir should be called `hostonly` but that's a different question.

Reply:
:-(

IMHO this is/was the most important part of this PR. Without this, the initramfs generated by the `50-dracut.install`  `kernel-install` module will **always** be `hostonly=yes` because of https://github.com/dracut-ng/dracut-ng/blob/main/dracut.conf.d/hostonly/50-hostonly.conf even if `/etc/dracut.conf` or `/etc/dracut.conf.d/*` specify `hostonly=no`. IMHO this is a bug.

Please consider bringing this change back.

Reply:
PS: My concrete use-case is this Debian package:

https://packages.debian.org/sid/all/dracut-config-generic/filelist

IMHO, installing it should ensure that all initramfs files generated by `dracut` are `hostonly=no` unless `dracut` was explicitly invoked by the user with `--hostonly`.

Reply:
@jozzsi I can understand your frustration. For what it's worth, I think the review by @mwilck lead to an ultimately better version of `test/TEST-20-STORAGE/test.sh`, which is a good thing.

Regarding `install.d/50-dracut.install`: @mwilck Is there maybe a misunderstanding here? Adding the `--add-confdir hostonly` option does **not** add a "config directory dedicated to running dracut from `install.d`". It adds a config directory which (as the name suggests) contains a config file (shipped by `dracut` upstream and installed by distros) that sets `hostonly=yes`. regardless of other configuration. I don't think that's intended, is it?

Reply:
> It adds a config directory which (as the name suggests) contains a config file (shipped by dracut upstream and installed by distros) that sets hostonly=yes.

Right. I agree that this is strange, in particular if there's no way to override this behavior. 

I was indeed confused. Yes I do agree that removing this line was the right thing to do in this PR.

Reply:
Thanks! :-)